gateway/CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Changelog for `twilight-gateway`.
 
+## [0.4.0] - 2021-05-??
+
+### Upgrade Path
+
+The MSRV is now Rust 1.49.
+
+Errors are no longer enums and don't expose their concrete underlying error
+source. You can access the underlying error via the implemented
+`std::error::Error::source` method or the `into_parts` or `into_source` methods
+on each error struct, which will return a boxed `std::error::Error`. To access
+the reason for the error use the `kind` or `into_parts` method on error structs;
+the returned error type is an enum with variants for each potential reason the
+error occurred.
+
+### Additions
+
+Add `rustls-webki-roots` feature to use WebPKI roots for RusTLS
```

Comment:
```suggestion
Add `rustls-webpki-roots` feature to use WebPKI roots for RustTLS
```

Reply:
The second part half of the suggestion isnt right. The library is called `rustls`, not rusttls.

Reply:
Oops, my bad